src/elixirServer.ts

```diff
@@ -116,6 +116,23 @@ export class ElixirServer {
         this.sendRequest('DEFL', command, resultCb);
     }
 
+    getDocumentation(document: vscode.TextDocument, position: vscode.Position, callback: Function): void {
+        const wordAtPosition = document.getWordRangeAtPosition(position);
+        const word = document.getText(wordAtPosition);
+        if (word.indexOf('\n') >= 0) {
+            console.error('[vscode-elixir] got whole file as word');
```

Comment:
This may have something to do with our word pattern in elixirConfiguration (wordPattern: /\w+[\.\w+]*/,) which may lead to document.getWordRangeAtPosition to sometimes return the wohle document... At the current time I don't have a solution for this.